docs/manifest.json

```diff
@@ -0,0 +1,260 @@
+{
+  "lang": "node",
+  "friendlyLang": "Node.js",
+  "markdown": "javascript",
+  "versions": [
```

Comment:
Maybe we should have a release script that:

1. Runs linter
2. Builds docs
3. Runs unit tests
4. Runs system tests
5. Makes release tag via `npm version`
6. Updates `manifest.json` & commits changes
7. Pushes (with tags) to master
8. Runs `npm publish`

--

Or, could the FE just ping GH's API for the versions? Maybe like how it currently gets the latest version for the homepage? `(Latest Release v0.27.0 Dec 22, 2015)`